monitoring/README.md

```diff
@@ -15,4 +15,222 @@ This directory contains chaos interleaved grafana dashboards to get started with
 
   ```
   cd grafana-dashboards
-  ```
\ No newline at end of file
+  ```
+
+- Utilities 
+
+  ```
+  cd utils
+  ```
```

Comment:
These are not necessary. Once the user enters the monitoring folder, the steps for execution can start.

Reply:
Okay removing the utility section from the readme.